board/safety/safety_gm.h

```diff
@@ -42,6 +53,18 @@ enum {
   GM_BTN_CANCEL = 6,
 };
 
+int gm_cam_bus = 2;
+
+enum {GM_OBD2, GM_CAM} gm_harness = GM_OBD2;
+bool gm_stock_long = false;
+
+#ifdef GM_EPS_TIMING_WORKAROUND
```

Comment:
Let's do all this stuff in another PR for now

Reply:
I didn't expect the timing stuff would be in the final pr - I'm hoping something can be done OP side.